### ISSUE TITLE
Force ipv6 for certain websites with ss-redir

### ISSUE THREAD

**Issue by dianlujitao** [shadowsocks/shadowsocks-libev#2178]
### What version of shadowsocks-libev are you using?
shadowsocks-libev 3.2.0

### What operating system are you using?
Server: Ubuntu 18.04
Client: openwrt

### What did you do?
It's widely known that google scholar banned v4 address of many popular vps providers such as vultr, linode. However, they provide v6 address survived from google's blacklist, which means that we can access google scholar through our vps by forcing ipv6 traffic.

Concretely, I added ipv6 address of google scholar to *server side* hosts, and I can successfully access to google scholar with ss-local, shadowsocks-qt5, shadowsocks-android, but not with ss-redir deployed on my openwrt router, the traffic still goes through ipv4.

I found a similar issue #1623 and the `disable_sni` option seems to be what I need, but it's removed now #1881

The question is, is it possible to route certain traffic to ipv6 in `dual stack server + v4 only client` environment?

### What did you expect to see?
ss-redir traffic are routed w.r.t hosts.

### What did you see instead?
ditto

### What is your config in detail (with all sensitive info masked)?
server side:
```
{
    "server":"0.0.0.0",
    "server_port":233333333,
    "password":"xxxxxxxxxxxxx",
    "timeout":60,
    "method":"chacha20-ietf-poly1305",
    "fast_open":true
}
```
client side: generated by luci-app-shadowsocks

**Comment by madeye** [comment]
Nope, you cannot do that. To redirect IPv6 traffic with `ss-redir`, you need to make sure your client has a valid IPv6 routing (at least for LAN) and properly setup IPv6 redirecting rules on your router.

**Comment by dianlujitao** [comment]
My LAN has IPv6, WAN is v4 only, are there any documentations showing how to redirect v6 traffic?

**Comment by madeye** [comment]
https://mirrors.bieringer.de/Linux+IPv6-HOWTO/nat-netfilter6..html

**Comment by dianlujitao** [comment]
I run `ss-redir` with server `::`, and netstat shows:
```
tcp        0      0 :::1234                 :::*                    LISTEN
```
then added the following rules:
```
ip6tables -t nat -I PREROUTING 1 -p tcp -j REDIRECT --to-ports 1234
ip6tables -t nat -I OUTPUT 1 -p tcp -j REDIRECT --to-ports 1234
```
but still no luck:
```
$ LANG=C wget --tries 1 http://ipv6.google.com
--2018-09-16 13:07:28--  http://ipv6.google.com/
Resolving ipv6.google.com (ipv6.google.com)... 2607:f8b0:4007:80c::200e
Connecting to ipv6.google.com (ipv6.google.com)|2607:f8b0:4007:80c::200e|:80... failed: Network is unreachable.
```
Is there still anything missing?

**Comment by madeye** [comment]
https://github.com/shadowsocks/shadowsocks-libev/pull/131
